### DISCUSSION TITLE
Changing the order of the list in hasMany fieldtype

### DISCUSSION THREAD

**Discussion by katwlodarczyk** [statamic-rad-pack/runway#228] [Feature Requests]
Hello, 
would it be possible to have the ability to order items in hasMany fieldtype via drag&drop, similar to ordering Entries in Statamic core?
![example](https://i.ibb.co/QpDHYn3/617-FE496-EC93-4-E3-C-9610-9-ED74-D366-B65.jpg)

Thanks!

**Comment by duncanmcclean** [comment]
Thanks for creating this feature request - it looks like it may already be possible when not using the Table 'mode' for the Has Many field.

I'll see what I can do though about making the Table 'mode' orderable.

References for when I come back to this:

* https://github.com/duncanmcclean/runway/blob/main/src/Fieldtypes/HasManyFieldtype.php#L112
* https://github.com/duncanmcclean/runway/blob/main/resources/js/components/Fieldtypes/Relationship/RelationshipTableMode.vue#L26

**Comment by duncanmcclean** [comment]
This feature has now been implemented in https://github.com/duncanmcclean/runway/pull/259 🎉
